tests/wsl/firstrun.pm

```diff
@@ -137,15 +144,23 @@ sub run {
         assert_screen 'wsl-image-ready-prompt', 120;
     }
 
-    unless (get_var('BETA')) {
-        become_root;
+    # Nothing to do in WSL2 pts w/o serialdev support
+    # https://github.com/microsoft/WSL/issues/4322
+    if (get_var('WSL2')) {
+        type_string "exit\n";
+        return;
     }
 
+    become_root unless (is_sut_dev() && is_sut_reg());
+
     assert_script_run 'cd ~';
     if (script_run "zypper ps") {
         record_soft_failure 'bsc#1170256 - [Build 3.136] zypper ps is missing lsof package';
     }
-
+    type_string "exit\n";
+    sleep 3;
```

Comment:
Last one. i believe you can use one of the parameters to avoid a sleep here. from the perlpod of type_string
"C<wait_still_screen> if set, C<wait_still_screen> returns true if screen is not                                                                                                                                                              
changed for given C<$wait_still_screen> seconds or false if the screen is not still                                                                                                                                                          
for the given seconds within defined C<timeout> after the whole string is typed.                                                                                                                                                             
Default timeout is 30s, default stilltime is 0s." WDYT?

Reply:
Let me try, hopefully I won't need to touch `similarity_level`

Reply:
Ok, flashing cursor makes the screen never still. I guess it can be solved by tweaking `similarity_level`, but I am not keen on doing so.

Reply:
no problem. Can you resolve this discussion for me?

Reply:
sure :)